src/cfnlint/custom_rules.py

```diff
@@ -4,37 +4,42 @@
 """
 # pylint: disable=W0108
 # pylint: disable=W0622
+
 import logging
 import cfnlint
+import cfnlint.runner
 import cfnlint.customRules.Operators
 import cfnlint.customRules.Rule
 
 LOGGER = logging.getLogger(__name__)
-Operator = {'EQUALS': lambda x, y: cfnlint.customRules.Operators.equalsOp(x, y),
-            'PLACEHOLDER': lambda x, y: LOGGER.debug('Placeholder Op')}
+Operator = {'EQUALS': lambda x, y, z: cfnlint.customRules.Operators.equalsOp(x, y, z),
+            'NOT_EQUALS': lambda x, y, z: cfnlint.customRules.Operators.notEqualsOp(x, y, z),
+            'INSET': lambda x, y, z: cfnlint.customRules.Operators.InSetOp(x, y, z),
+            'NOT_INSET': lambda x, y, z: cfnlint.customRules.Operators.NotInSetOp(x, y, z)}
```

Comment:
thoughts on 
```suggestion
            'IN': lambda x, y, z: cfnlint.customRules.Operators.InSetOp(x, y, z),
            'NOT_IN': lambda x, y, z: cfnlint.customRules.Operators.NotInSetOp(x, y, z)}
```

Reply:
The current structure of the program compares the raw operator input from the text file with one of the dict keys - I would prefer to keep that portion as is, as it's one of the neatest ways I think that could be coded.

On the other hand, I do get what you mean by there being a lot of duplicate code with the operators. I think another approach I would look into is creating a general helper class instead, and having each operator run that if possible for the duplicate portions.

Reply:
> The current structure of the program compares the raw operator input from the text file with one of the dict keys - I would prefer to keep that portion as is, as it's one of the neatest ways I think that could be coded.

Will edit to clarify. I agree with this, but was just wondering about your thoughts on alternative operator syntax for the set operators themselves

> On the other hand, I do get what you mean by there being a lot of duplicate code with the operators. I think another approach I would look into is creating a general helper class instead, and having each operator run that if possible for the duplicate portions.

even just a method called in the methods in `src/cfnlint/customRules/Operators.py` could help if another class seems like overkill

Reply:
Ooooh, I misunderstood. That new syntax does sound a lot better!

Yeah, I'm thinking one common helper method within the operators itself would make the whole file a whole lot neater with so much duplicate code removed.